scripts/build/windows.bat

```diff
@@ -226,6 +233,10 @@ signtool verify /pa /v %package_output%\Etcher.exe
 
 upx -9 %package_output%\*.dll
 
+cd %package_output%^
+ && 7z a -tzip ..\installers\%package_name%.zip *^
```

Comment:
> their web page is ugly and unkempt to the point where I still wouldn't download it unless forced to.

I guess some people prefer working on improving their software, to working on prettifying their website? ;-)

> I'd contend that people shouldn't be on pre-Windows 8 versions anyway

My Windows laptop is still chugging away quite happily on Win7.

> Honestly, it might make sense for you to take the binaries for as many of Etcher's build deps as possible and push them to a server so that they could then be downloaded during the build, greatly simplifying the work on a contributor.

IIRC that's kinda the idea behind [Vagrant](https://www.vagrantup.com/) - but as you say, a larger-scale project.